test_collections/sdk_tests/__init__.py

```diff
@@ -13,4 +13,5 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+from .support.python_testing import sdk_python_collection
```

Comment:
do we also need a custom_python_collection?

Reply:
I think it might make sense.I filled an issue for that: https://github.com/project-chip/certification-tool/issues/97